### ISSUE TITLE
contributing guide: ran into an issue while installing tests

### ISSUE THREAD

**Issue by ElijahQuiazon** [django/channels#2012]
Followed the contribution [guide](https://github.com/django/channels/blob/main/CONTRIBUTING.rst#quick-setup) and faced an error when installing the tests.

**Specs**
* Python 3.9.12
* Django 4.2.1
* zsh 5.8.1

**Steps to reproduce**
1. Fork `django/channels` repo
2. Clone repo into local machine with `git clone git@github.com:your-username/channels.git`
3. Activate virtual environment
4. Move into channels directory with `cd channels`
5. Install tests with `python -m pip install -e .[tests]`

**Error**
`zsh: no matches found: .[tests]`

After some research, I identified that the `[]` might have been interpreted by zsh as a wildcard to match files similar to `tests`. After wrapping it around quotations (i.e `'[tests]'`), I managed to succeed with installations.

Not sure if it's just an isolated issue — just thought to raise it up in case it's worth mentioning.

**Comment by carltongibson** [comment]
Yes it's a zsh vs bash thing. You could suggest a change there if you'd like! Thanks.

**Comment by ElijahQuiazon** [comment]
Sure thing, I've raised a PR regarding this. Sorry it took awhile — was swamped recently.
